{{ .ProjectSnake }}/.bazelrc

```diff
@@ -3,6 +3,11 @@ try-import %workspace%/tools/java17.bazelrc
 
 ### YOUR PROJECT SPECIFIC OPTIONS GO HERE ###
 
+{{- if .Computed.go }}
+# See https://github.com/bazelbuild/bazel/issues/20186
```

Comment:
Maybe expand the comment to explain why we're adding it? Is it required?

The link points to an issue titled "Figure out a plan ..." which doesn't really explain why we would need or want this...

Reply:
We don't technically need it. It's recommended by https://github.com/bazel-contrib/rules_go/blob/master/docs/go/core/bzlmod.md#depending-on-tools but if @fmeum thinks it's too experimental to turn on in presets, we shouldn't turn on here either I think.